bazel_tools/scala_version.bzl

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
This file is very similar (in both shape and purpose) to `bazel_tools/build_environment.bzl`. Would it make sense to merge them?

Reply:
Not quite sure, they are used for very different things and half of the logic in there is for converting from env var to Bazel var which works completely differently between the two. Given how stable `build_environment.bzl` has been, I’m tempted to keep them separate for now.